utils/spdx/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -33,8 +33,7 @@ private const val SPDX_VERSION_MAJOR_MINOR = "SPDX-2.2"
 private val DATA_LICENSE = SpdxLicense.CC0_1_0.id
 
 /**
- * An SPDX document as specified by https://github.com/spdx/spdx-spec/tree/development/v2.2.1/chapters and
- * https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/ in revision 947271b.
+ * An SPDX document as specified by https://spdx.github.io/spdx-spec/v2.2.2/.
```

Comment:
Does the implementation implement 2.2.1 or 2.2.2 ?

Reply:
AFAIK there is no implementation-wise differences between these versions. Also the `spdxVersion` field we write out is major / minor only, omitting the patch level, see

https://github.com/oss-review-toolkit/ort/blob/8b346d177342b5abd854b0f33a575485d27686c1/utils/spdx/src/main/kotlin/model/SpdxDocument.kt#L48-L51